Index: src/main/java/com/gamjaring/web/springboot/friendedit/FriendeditRepository.java
===================================================================
diff --git a/src/main/java/com/gamjaring/web/springboot/friendedit/FriendeditRepository.java b/src/main/java/com/gamjaring/web/springboot/friendedit/FriendeditRepository.java
--- a/src/main/java/com/gamjaring/web/springboot/friendedit/FriendeditRepository.java	
+++ b/src/main/java/com/gamjaring/web/springboot/friendedit/FriendeditRepository.java	
@@ -1,2 +1,6 @@
-package com.gamjaring.web.springboot.friendedit;public interface FriendeditRepository {
+package com.gamjaring.web.springboot.friendedit;
+
+public interface FriendeditRepository {
+
+
 }
Index: src/main/java/com/gamjaring/web/springboot/friendedit/Friendedit.java
===================================================================
diff --git a/src/main/java/com/gamjaring/web/springboot/friendedit/Friendedit.java b/src/main/java/com/gamjaring/web/springboot/friendedit/Friendedit.java
--- a/src/main/java/com/gamjaring/web/springboot/friendedit/Friendedit.java	
+++ b/src/main/java/com/gamjaring/web/springboot/friendedit/Friendedit.java	
@@ -1,2 +1,30 @@
-package com.gamjaring.web.springboot.friendedit;public class friendedit {
+package com.gamjaring.web.springboot.friendedit;
+
+import com.gamjaring.web.springboot.user.User;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name="friendedit")
+@Entity
+public class Friendedit {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column
+    private User user;
+
+
+
+
 }
Index: src/main/java/com/gamjaring/web/springboot/friendedit/Imgfile.java
===================================================================
diff --git a/src/main/java/com/gamjaring/web/springboot/friendedit/Imgfile.java b/src/main/java/com/gamjaring/web/springboot/friendedit/Imgfile.java
--- a/src/main/java/com/gamjaring/web/springboot/friendedit/Imgfile.java	
+++ b/src/main/java/com/gamjaring/web/springboot/friendedit/Imgfile.java	
@@ -1,2 +1,39 @@
-package com.gamjaring.web.springboot.friendedit;public class Imgfile {
+package com.gamjaring.web.springboot.friendedit;
+
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+//@AllArgsConstructor   4가지 속성 이미 생성자 되있어서 노필요
+@Table(name="imgfile")
+@Entity
+public class Imgfile {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String origFilename;
+
+    @Column(nullable = false)
+    private String filename;
+
+    @Column(nullable = false)
+    private String filePath;
+
+    @Builder
+    public Imgfile(Long id, String origFilename, String filename, String filePath) {
+        this.id = id;
+        this.origFilename = origFilename;
+        this.filename = filename;
+        this.filePath = filePath;
+    }
+
+
+
 }
Index: src/main/java/com/gamjaring/web/springboot/friendedit/FriendService.java
===================================================================
diff --git a/src/main/java/com/gamjaring/web/springboot/friendedit/FriendService.java b/src/main/java/com/gamjaring/web/springboot/friendedit/FriendService.java
--- a/src/main/java/com/gamjaring/web/springboot/friendedit/FriendService.java	
+++ b/src/main/java/com/gamjaring/web/springboot/friendedit/FriendService.java	
@@ -1,2 +1,4 @@
-package com.gamjaring.web.springboot.friendedit;public class FriendService {
+package com.gamjaring.web.springboot.friendedit;
+
+public class FriendService {
 }
Index: src/main/java/com/gamjaring/web/springboot/friendedit/ImgfileDto.java
===================================================================
diff --git a/src/main/java/com/gamjaring/web/springboot/friendedit/ImgfileDto.java b/src/main/java/com/gamjaring/web/springboot/friendedit/ImgfileDto.java
--- a/src/main/java/com/gamjaring/web/springboot/friendedit/ImgfileDto.java	
+++ b/src/main/java/com/gamjaring/web/springboot/friendedit/ImgfileDto.java	
@@ -1,2 +1,38 @@
-package com.gamjaring.web.springboot.friendedit;public class ImgfileDto {
-}
+package com.gamjaring.web.springboot.friendedit;
+
+import lombok.*;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+public class ImgfileDto {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String origFilename;
+    private String filename;
+    private String filePath;
+
+    public Imgfile toEntity() {
+        Imgfile build = Imgfile.builder()
+                .id(id)
+                .origFilename(origFilename)
+                .filename(filename)
+                .filePath(filePath)
+                .build();
+        return build;
+    }
+
+    @Builder
+    public ImgfileDto(Long id, String origFilename, String filename, String filePath) {
+        this.id = id;
+        this.origFilename = origFilename;
+        this.filename = filename;
+        this.filePath = filePath;
+    }
+}
\ No newline at end of file
Index: src/main/java/com/gamjaring/web/springboot/friendedit/ImgfileRepository.java
===================================================================
diff --git a/src/main/java/com/gamjaring/web/springboot/friendedit/ImgfileRepository.java b/src/main/java/com/gamjaring/web/springboot/friendedit/ImgfileRepository.java
--- a/src/main/java/com/gamjaring/web/springboot/friendedit/ImgfileRepository.java	
+++ b/src/main/java/com/gamjaring/web/springboot/friendedit/ImgfileRepository.java	
@@ -1,2 +1,6 @@
-package com.gamjaring.web.springboot.friendedit;public interface ImgfileRepository {
+package com.gamjaring.web.springboot.friendedit;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ImgfileRepository extends JpaRepository<Imgfile, Long> {
 }
Index: src/ .travis.yml
===================================================================
diff --git a/src/ .travis.yml b/src/ .travis.yml
--- a/src/ .travis.yml	
+++ b/src/ .travis.yml	
@@ -1,0 +1,20 @@
+language: java
+jdk:
+  - openjdk8
+
+branches: # 1.
+  only:
+    - master
+
+# Travis CI 서버의 Home
+cache: # 2.
+  directories:
+    - '$HOME/.m2/repository'
+    - '$HOME/.gradle'
+script: "./gradlew clean build" # 3.
+
+# CI 실행 완료 시 메일 알람
+notifications: # 4.
+  email:
+    recipients:
+      - qweadzs@naver.com
\ No newline at end of file
